divide README in several files (#5)
Pix4D/ntrip-catalog: CONTRIBUTING.md

# Contributing

## Who can contribute?

> Anybody is open to contribute to this catalog, but ideally we expect NTRIP providers to submit their data themselves.

Reviewer: ```suggestion
Anybody is welcome to contribute to this catalog, but ideally we expect NTRIP providers to submit their data themselves.
```

If you find the proper documentation published by the NTRIP provider confirming the CRS used, and do not expect the provider to do it, you can contribute it as well.
Notice that any contribution should be properly documented, filling the `reference` section in the json file.

## How to contribute
Just create a PR in this GitHub project.
In case you do not know how to do it, or you do not feel comfortable with JSON, you can open an Issue providing all the data.
All contributions must be properly documented, filling the `reference` section in the JSON file.
This database is intended to become a trusted source of information,

> so submissions that cannot be backed by an authoritative reference will not be considered.

Reviewer: ```suggestion
therefore submissions that cannot be backed by an authoritative reference will not be considered.
```

> See the file in the `data` directory to see what can be done.

Reviewer: what do you mean with this?

> See the file in the `data` directory to see what can be done.

Reviewer: ```suggestion
Any new entry should be added to the appropriate country/region-specific file below the `data` folder. Existing entries in this folder can be used as an example and starting point.
```
